remote-api/src/test/java/org/alfresco/rest/api/tests/AlfrescoCMISBasicsTestGroup.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * #%L
+ * Alfresco Remote API
+ * %%
+ * Copyright (C) 2005 - 2022 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.rest.api.tests;
+
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.apache.chemistry.opencmis.client.api.CmisObject;
+import org.apache.chemistry.opencmis.client.api.Property;
+import org.apache.chemistry.opencmis.tck.CmisTestResult;
+import org.apache.chemistry.opencmis.tck.tests.basics.BasicsTestGroup;
+import org.apache.chemistry.opencmis.tck.tests.basics.RootFolderTest;
+
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.when;
+
+class AlfrescoCMISBasicsTestGroup extends BasicsTestGroup
+{
+    @Override
+    public void init(Map<String, String> parameters) throws Exception
+    {
+        super.init(parameters);
+
+        replaceRootFolderTest();
+    }
+
+    private void replaceRootFolderTest() throws Exception {

```

Comment:
Replacing test by using the fact that getTests() method returns actual modifiable list reference